Libplanet/Net/Swarm.cs

```diff
@@ -693,6 +693,8 @@ await DialToExistingPeers(dialTimeout, cancellationToken)
                 return;
             }
 
+            peersWithHeight = peersWithHeight.OrderByDescending(p => p.Item2).ToList();
```

Comment:
How about sorting it before it's become realized (i.e., before `.ToListAsync()` being called) rather than here?

Reply:
I tried to but couldn't. Maybe because `peersWithHeight` is `IAsyncEnumerable`.